### PR TITLE
[CAP-212] Create product form

### PR DESCRIPTION
**Pull request by lpelizzaMS**
### Background
Create a new screen where the user can create a new product.

### Screenshot
rn-createproduct

### CODE REVIEW

File: IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/ViewModels/LoginViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+class LoginViewModel {
+    
+    var email = Observable<String?>("")
+    var password = Observable<String?>("")
+    
+    func login()  {
+        // TODO: Implement dummy async login
+    }
+    
+}
```
Comment (mravinale) [inline IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/ViewModels/LoginViewModel.swift]: do we need any .swift files?

**Comment by mvazquez-ms** [comment]
Are you sure this PR is ready to be reviewed? I'm seeing conflicts, files that should be ignored and files that have been taken from another project (based on auto-generated headers).

Let us know when that's fixed. Also, make sure you use `spaces` and not `tabs.`

**Comment by mravinale** [comment]
please lucas lets follow the javascript convention https://github.com/MakingSense/code-style-guides/tree/master/React and use spaces @lpelizzaMS

**Comment by lpelizzaMS** [comment]
@mvazquez-ms 
I just created a new PR, using spaces instead of tabs.
https://github.com/makingsensetraining/mobile-pocs/pull/34

**Comment by mravinale** [comment]
Thiank for updating the tabs vs spaces but javascript / react conventions requires only 2 spaces
https://github.com/MakingSense/code-style-guides/tree/master/React sorry that I didn't mention before

**Comment by lpelizzaMS** [comment]
WoW, I didn't know that. But I just updated the number of spaces.

**Pull request merged**